docs/HACKING.md

```diff
@@ -139,4 +139,26 @@ opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
 and removed from it:
 ```shell script
 opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
+```
+
+## Updating Git submodules
```

Comment:
I should have caught this before. I realize this is for updating the submodules after a pull. What we need to add here is how to actually update the submodules from it's respective upstream.